cmd/dextre/cmd/robots_gen_gc.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// robotsGenGCCmd represents the robots gen_gc command
+var robotsGenGCCmd = &cobra.Command{
+	Use:   "gen_gc",
```

Comment:
Why use such a weird name? Why not call it `generate-golden` or something more descriptive?

Reply:
ok